java-client/src/integration-test/java/com/cumulocity/sdk/client/measurement/MeasurementIT.java

```diff
@@ -648,8 +656,10 @@ private void iShouldGetNumberOfMeasurements(int count) {
         assertThat(collection1.getMeasurements()).hasSize(count);
     }
 
-    private void iShouldEventuallyGetNumberOfMeasurements(int count) {
-        await().atMost(TEN_SECONDS).until(() -> measurementApi.getMeasurements().get().getMeasurements().size() == count);
+    private void iShouldEventuallyGetNumberOfMeasurements(int count, int moIndex) {
```

Comment:
```suggestion
    private void iShouldEventuallyGetNumberOfMeasurementsForSource(int count, int moIndex) {
```
or maybe split into fancy builder(s) or maybe pass MORepresentation as parameter:
```suggestion
    private void iShouldEventuallyGetNumberOfMeasurements(int count, ManagedObjectRepresentation source) {
```

What I want to improve is calling this method with 2 numeric parameters with totally different meanings